ext/openssl/php_openssl.h

```diff
@@ -86,6 +94,13 @@ php_stream_transport_factory_func php_openssl_ssl_socket_factory;
 
 void php_openssl_store_errors();
 
+PHP_OPENSSL_API int php_openssl_cipher_iv_length(char *method, zend_long *ret);
+PHP_OPENSSL_API int php_openssl_random_pseudo_bytes(zend_string **buffer, zend_long length);
+PHP_OPENSSL_API int php_openssl_encrypt(char *data, size_t data_len, char *method, size_t method_len, char *password,
```

Comment:
Again returning output seems better

Reply:
Why do tests fail more?

It's just replacing
```
if (php_openssl_encrypt(data, ..., &ret) == SUCCESS)
```
with
```
if ((ret = php_openssl_encrypt(data, ...)) != NULL)
```

Reply:
Yes, found out was due to few return statements handling differences ... should work better now.